entity-framework/core/index.md

```diff
@@ -10,21 +10,28 @@ uid: core/index
 
 Entity Framework (EF) Core is a lightweight, extensible, [open source](https://github.com/aspnet/EntityFrameworkCore) and cross-platform version of the popular Entity Framework data access technology.
 
-EF Core can serve as an object-relational mapper (O/RM), enabling .NET developers to work with a database using .NET objects, and eliminating the need for most of the data-access code they usually need to write.
+EF Core can serve as an object-relational mapper (O/RM), which:
+
+* Enables .NET developers to work with a database using .NET objects.
+* Eliminates the need for most of the data-access code that typically needs to be written.
 
 EF Core supports many database engines, see [Database Providers](xref:core/providers/index) for details.
 
 ## The Model
```

Comment:
We should standardize to sentence case for non-title headings

Reply:
Per MS style guide, use sentence case.